### ISSUE TITLE
Arducam Mini Shield ov2640 SDCard error on Arduino and Esp12E

### ISSUE THREAD

**Issue by ayoadejoe** [ArduCAM/Arduino#194]
Hello, I successfully set up arducam mini with Arduino and took shots with arducam.exe. I also successfully used it with esp12e/esp8266 video streaming and capture via browser.
The issue is, I need to make it work with SDCard. I used different Chip select as expected, made sure both are not low at the same time, nevertheless it keeps returning interface error.
I used same mosi, miso, clk  but used different cs on both uno and 8266, same error.
How can I resolve this?

**Comment by supprot** [comment]
@ayoadejoe 
Hi,
What's your  SD CS?
Regards,

**Comment by ayoadejoe** [comment]
@supprot 
On Arduino uno, I used pin 7 for arducam and 4 for a sdcard 
On esp, I used gpio 16 (D1) for arducam and gpio 2 (D4) for sdcard

Both yielded the same error. However, whenever I remove the miso/mosi pin, the arducam runs but sdcard fails (of course).

**Comment by ayoadejoe** [comment]
Hello, I successfully set up arducam mini with Arduino and took shots with arducam.exe. I also successfully used it with esp12e/esp8266 video streaming and capture via browser.
The issue is, I need to make it work with SDCard. I used different Chip select as expected, made sure both are not low at the same time, nevertheless it keeps returning interface error.
I used same mosi, miso, clk but used different cs on both uno and 8266, same error.
How can I resolve this?
